src/components/card/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, ReactNode } from 'react';
+
+import { Box, chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+import Stack from '../stack';
+
+interface Props {
+  renderImage: () => ReactNode;
+  carTitle: string;
+  price: string;
+  dealerName: string;
+  dealerAddress: string;
+}
+
+const Card: FC<Props> = ({
+  renderImage,
```

Comment:
Do we need to use a function here or can we just pass the node to render instead? (We don't pass any parameters to the renderer from the card...)

Reply:
we can just pass the node. Thanks!